docs/source/morpheus_quickstart_guide.md

```diff
@@ -1137,6 +1085,19 @@ This section lists solutions to problems you might encounter with Morpheus or fr
 
   - Solution: Reinstall the Morpheus workflow and reduce the Kafka topic's message retention time and message producing rate.
 
+### The dropna stage
```

Comment:
@raykallen @gbatmaz @tzemicheal @bsuryadevara can you check me on these details pls?

Reply:
they all look correct to me.
can you add --
| DFP Cloudtrail | userIdentitysessionContextsessionIssueruserName |